.travis.yml

```diff
@@ -10,7 +10,7 @@ matrix:
     - env: variant=debug crypto=openssl docs=1 tests=1 examples=1 tools=1 toolset=darwin
       os: osx
       osx_image: xcode11.2
-    - env: variant=debug crypto=openssl python=1 toolset=darwin ios=1
+    - env: variant=debug crypto=openssl webtorrent=on python=1 toolset=darwin ios=1
```

Comment:
unfortunately this isn't enough to enable webtorrent builds. this will set an environment variable.
On line 265 you also need something like this:
```
  - 'if [[ $websocket == "on" ]]; then
      export B2_ARGS="${B2_ARGS} websocket=on";
    fi'
```

Reply:
Oh thanks, I was wondering why I wasn't working.